credit-risk/src/predictor.py

```diff
@@ -1,15 +1,13 @@
 import h2o
 import pandas as pd
 
-from h2o.estimators.gbm import H2OGradientBoostingEstimator
+from h2o_wave_ml import build_model, ModelType
 
 
-class Predictor:
-    """
-    Wrapper for H2O-3
 
-    ChurnPredictor builds an abstraction between H2O-3 machine learning library and the Churn Risk app
-    giving the developer freedom to integrate any 3rd party machine library with a minimal change to the app code.
+class ChurnPredictor:
+    """
+    ChurnPredictor builds an abstraction between WaveML library and the Churn Risk app.
```

Comment:
Ah, I think this is an old copy-paste error, so this app should be either `Predictor` or `CreditPredictor` if it needs to be specific

Reply:
Ya I missed it too, been looking to so many apps :-)